### PR TITLE
Update example path to dart sdk w/ homebrew path

### PR DESCRIPTION
**Pull request by legalcodes**

### CODE REVIEW

File: src/tools/jetbrains-plugin.md [hunk 1]
```diff
@@ -89,8 +89,8 @@ Here's one way to configure Dart support:
   </p>
 
   <p>
-    For example, the SDK path might be
-    <code><em>&lt;dart installation directory></em>/dart/dart-sdk</code>.
+    For example, the SDK path might be (if you installed via homebrew)
```
Comment (kwalrath) [inline src/tools/jetbrains-plugin.md]: I don't think the majority of users are using macOS. (Analytics should be able to tell us.) Maybe we should show two or three common paths?

Comment (mit-mit) [reply src/tools/jetbrains-plugin.md]: Actually, the majority (~45%) are on Windows. Then macOS, and then a small share on Linux.

Comment (kwalrath) [reply src/tools/jetbrains-plugin.md]: Do we know the path that the majority of Windows users are likely to have?

Comment (mit-mit) [reply src/tools/jetbrains-plugin.md]: Sorry, no, I think that varies a lot depending on how they install it.

Comment (kwalrath) [reply src/tools/jetbrains-plugin.md]: @legalcodes maybe you could ask on Gitter?

Comment (legalcodes) [reply src/tools/jetbrains-plugin.md]: Great suggestion re: showing multiple common paths. Will find out the most common path for Windows setup. Thanks!

**Comment by kwalrath** [comment]
@legalcodes were you able to get any Windows details?

**Comment by legalcodes** [comment]
I was able to get a Windows machine -- will go through the install flow and update this issue.

**Comment by mit-mit** [comment]
Is this still being worked on?

**Comment by legalcodes** [comment]
Yes, though windows setup is taking longer than expected. Will keep this issue updated.

**Comment by kwalrath** [comment]
@legalcodes any progress, or should I close this?

**Pull request closed without merging**